### PR TITLE
[BUGFIX] .htaccess: Add file

### PR DESCRIPTION
**Pull request by M-arcus**
Fixes https://github.com/friendica/friendica/pull/5042#issuecomment-389218702

### CODE REVIEW

**Comment by M-arcus** [comment]
Not sure about the .gitignore though

**Comment by M-arcus** [comment]
@tobiasd Done. Also the .gitignore was edited.

**Comment by MrPetovan** [comment]
Thanks for your reactivity!

**Comment by tobiasd** [comment]
The README file in develop seems to need a correction as well.

**Comment by Quix0r** [comment]
You are doing the wrong way! This way, no local changes can be done. It was intension to rename it to `.htaccess-dist` so you copy it locally. Very sad about this "fix". :-1:

**Comment by MrPetovan** [comment]
This is the problem of having a PR too big, I didn't want to review everything again, which means that I missed the unrelated features added by your rebase.

**Comment by Quix0r** [comment]
OT/.htaccess "debacle" : I have now recreated my change and made another branch for it. This change also involves `doc/Install.md`.

**Comment by Quix0r** [comment]
Sure, I can understand that. :-/ And @M-arcus  was just wondering where the file is now.

**Comment by tobiasd** [comment]
There were no changes to `doc/Install.md` in #5042 as far as I see.

**Comment by Quix0r** [comment]
With my upcoming branch, this will be back in a similar way.

**Comment by tobiasd** [comment]
So what is your mitigation strategy for existing nodes? There needs to be an automatic strategy to not move the exiting file.

Breaking existing installation is not an option.

File: .gitignore [hunk 1]
```diff
@@ -62,6 +62,3 @@ vendor/*
 
 #ignore addons/ directory
 addons/
-
```
[the remark this answers is not in the document]
Comment (tobiasd) [reply .gitignore]: Without a hint about that, nope then you are sitting in front of a monitor with a broken webpage and this is a no-go!

**Comment by M-arcus** [comment]
@Quix0r Sorry, my automatic builds were failing because the .htaccess suddenly did not exist anymore (because it was moved)

I agree with your idea of having a `.htaccess-dist`, but there needs to be some migration for existing installations, like @tobiasd said

**Comment by MrPetovan** [comment]
The basic principle is that the `.htaccess-dist` has to be added in a first commit, and then the `.htaccess` needs to be added to `.gitignore` in a second commit. This way git doesn't figure it's a deletion + creation, a.k.a a move.

But the install should also take care of copying the `.htaccess-dist` to `.htaccess` and if it can't then documentation should be added to warn admins.

**Comment by Quix0r** [comment]
Then I should re-commit my branch when it is time.

**Pull request merged**